WordPress/Classes/ViewRelated/Media/MediaLibraryMediaPickingCoordinator.swift

```diff
@@ -90,7 +89,12 @@ final class MediaLibraryMediaPickingCoordinator {
     }
 
     private func showStockPhotos(origin: UIViewController, blog: Blog) {
-        stockPhotos.presentPicker(origin: origin, blog: blog)
+        let picker = StockPhotosPicker()
+        // add delegate conformance, allow release of picker in the same manner as the tenor picker
+        // in order to prevent duplicated uploads and botched de-selection on second upload
```

Comment:
```suggestion
```

Reply:
I think we don't necessarily need the comment in this instance as it's specific to the bug we're fixing :)

Reply:
Hmm, I was hoping I could commit this change via Github, but I don't think I can because it's on a fork. No worries, I'll go ahead and merge as-is.